lib/analyzer/index.ts

```diff
@@ -0,0 +1,26 @@
+import * as osReleaseDetector from './os-release-detector';
+import * as imageIdDetector from './image-id-detector';
+import * as apkAnalyzer from './apk-analyzer';
+import * as aptAnalyzer from './apt-analyzer';
+import * as rpmAnalyzer from './rpm-analyzer';
+
+export {
+  analyze,
+};
+
+function analyze(targetImage) {
+  return Promise.all([
+    imageIdDetector.detect(targetImage),
+    osReleaseDetector.detect(targetImage),
+    Promise.all([
+      apkAnalyzer.analyze(targetImage),
+      aptAnalyzer.analyze(targetImage),
+      rpmAnalyzer.analyze(targetImage),
+    ]),
+  ])
+  .then(res => ({
+    imageId: res[0],
+    osRelease: res[1],
+    results: res[2],
```

Comment:
in the GO code, on a happy flow, the size of the results arr is only 1 (I'm pretty sure), but now it will always be 3. Is it OK with the output later on?

Reply:
> in the GO code, on a happy flow, the size of the results arr is only 1 (I'm pretty sure)

Nope :)

> Is it OK with the output later on?

Yup, because we already deal with it